app/views/registrations/_new.erb

```diff
@@ -0,0 +1,77 @@
+<div class="modern-design">
+  <main class="small-container">
+    <header>
+      <h1><%= t("registrations.new.title") %></h1>
+      <p><%= t("registrations.new.subtitle", podname: AppConfig.settings.pod_name, tutorials_link: link_to(t('registrations.new.tutorials_link_label'), "https://diasporafoundation.org/getting_started/sign_up", target: "_blank")).html_safe %></p>
+    </header>
+    <%= form_for(resource, url: registration_path(resource_name)) do |f| %>
+      <div class="flex-container fields">
+        <div>
+          <section>
+            <p>
+              <label for="user_username"><%= t("registrations.new.username") %></label>
+            </p>
+            <p>
+              <input type="text" id="user_username" name="user[username]" value="<%= @user.username %>" class="form-control" pattern="[A-Za-z0-9_.\-]+" required />@<%= AppConfig.pod_uri.host %>
```

Comment:
Not a problem on most production pods, but I just noticed that on my local dev-system, this doesn't include the port, which would also be part of the diaspora-ID. What you want to use instead is `AppConfig.bare_pod_uri`, or maybe even `User.diaspora_id_host` (which also includes the `@`).